src/controllers/auth.controller.ts

```diff
@@ -42,33 +42,29 @@ class AuthController {
           };
         });
 
-        const errorResponse = {
+        response.status(400).json({
           success: false,
           status: 400,
           message: 'Invalid parameters',
           data: errorPayload,
-        };
-
-        response.status(400).json(errorResponse);
-      } else {
-        const e = error as Error;
-        userResponse = {
-          success: false,
-          status: 500,
-          message: 'An unexpected error occurs.',
-        };
-        response.status(500).json(userResponse);
-        logger.error(e.message);
+        });
       }
+      
+      response.status(500).json({
+        success: false,
+        status: 500,
+        message: 'An unexpected error occurs.',
+      });
+      
+      logger.error((error as Error).message);
     }
   };
```

Comment:
No return from this code bloc, then the request could create an ISE.

Reply:
My bad